manager.go

```diff
@@ -686,11 +686,11 @@ func (m *jobManager) resolveImageOrVersion(imageOrVersion, defaultImageOrVersion
 			}
 			return "", "", fmt.Errorf("no stable, official prerelease, or nightly version published yet for %s", imageOrVersion)
 		} else if unresolved == "nightly" {
-			unresolved = "4.7.0-0.nightly"
+			unresolved = "4.8.0-0.nightly"
```

Comment:
Let's leave nightly and ci at 4.7 for the next week or so, and then we can potentially improve this so we can control more granularly

Reply:
Ok, let's keep it as is. Updated.